pkg/kubelet/dockertools/manager.go

```diff
@@ -1803,7 +1803,7 @@ func (dm *DockerManager) SyncPod(pod *api.Pod, runningPod kubecontainer.Pod, pod
 		}
 
 		// TODO(dawnchen): Check RestartPolicy.DelaySeconds before restart a container
-		namespaceMode := fmt.Sprintf("container:%v", podInfraContainerID)
+		namespaceMode := getIPCMode(pod, fmt.Sprintf("container:%v", podInfraContainerID))
```

Comment:
Won't this cause `hostIPC` configured pods to send "host" for the netMode which may be incorrect?